publish.sh

```diff
@@ -88,12 +85,32 @@ if [ "$PROJECT" == "orm" ] || [ "$PROJECT" == "reactive" ]; then
 	  EXTRA_ARGS=" --dry-run"
 	fi
 
-  ./gradlew releasePerform closeAndReleaseSonatypeStagingRepository -x test \
-    --no-scan --no-daemon --no-build-cache --stacktrace $EXTRA_ARGS \
-    -PreleaseVersion=$RELEASE_VERSION -PdevelopmentVersion=$DEVELOPMENT_VERSION \
-    -PdocPublishBranch=production -PgitRemote=origin -PgitBranch=$BRANCH
+	if [ -f "./jreleaser.yml" ]; then
+		# JReleaser-based build
+		export JRELEASER_GPG_HOMEDIR="$RELEASE_GPG_HOMEDIR"
+		# Get the jreleaser downloader
+		curl -sL https://git.io/get-jreleaser > get_jreleaser.java
+		# Download JReleaser with version = <version>
+		# Change <version> to a tagged JReleaser release
+		# or leave it out to pull `latest`.
+		java get_jreleaser.java
+		# Let's check we've got the right version
+		java -jar jreleaser-cli.jar --version
+		# Execute a JReleaser command such as 'full-release'
+		java -jar jreleaser-cli.jar full-release -Djreleaser.project.version="$RELEASE_VERSION"
+
+		./gradlew completeRelease -x test \
```

Comment:
Can you clarify what this is expected to achieve?

I'm wondering what happens if this fails after jreleaser completed the release.

Reply:
This one is creating a tag, resetting the version to snapshot and pushing it all to git.
(for ORM/Reactive though ... it probably will have to be the one that also uploads the documentation)

Reply:
> This one is creating a tag, resetting the version to snapshot and pushing it all to git.

We actually have SH scripts that do exactly that... Maybe leverage them? See `update-version.sh` and `push-upstream.sh`. Note we use that for HCANN, so it works on Gradle too.

>  (for ORM/Reactive though ... it probably will have to be the one that also uploads the documentation)

Ah. I guess it will. I thought jreleaser could handle such things, but given we want to move to https://github.com/hibernate/docs.hibernate.org/ eventually... it's probably best if we wait a bit before we investigate this.

Reply:
oh yeah and with ORM ... there's also the gradle plugin portal that we could only publish through gradle..